eclass/dotnet-utils.eclass

```diff
@@ -0,0 +1,216 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+# @ECLASS: dotnet-utils.eclass
+# @MAINTAINER:
+# anna@navirc.com
+# @AUTHOR:
+# Anna Figueiredo Gomes <anna@navirc.com>
+# @SUPPORTED_EAPIS: 7 8
+# @BLURB: common functions and variables for dotnet builds
+
+case ${EAPI} in
+	7|8) ;;
+	*) die "${ECLASS}: EAPI ${EAPI} unsupported." ;;
+esac
+
+if [[ ! ${_DOTNET_UTILS_ECLASS} ]]; then
+_DOTNET_UTILS_ECLASS=1
+
+inherit edo multiprocessing
+
+# @ECLASS_VARIABLE: DOTNET_SLOT
+# @REQUIRED
+# @PRE_INHERIT
+# @DESCRIPTION:
+# Allows to choose a slot for dotnet
+
+if [[ ! ${DOTNET_SLOT} ]]; then
+	die "${ECLASS}: DOTNET_SLOT not set"
+fi
+
+# Temporary, use the virtual once you can have multiple virtuals installed at once
```

Comment:
Do we have a bug filed for whatever the issue is here?

Reply:
https://bugs.gentoo.org/882569#c1